fiboa_cli/datasets/fs_br.py

```diff
@@ -0,0 +1,206 @@
+from ..convert_utils import convert as convert_
+import re
+import pandas as pd
+
+# Please REMOVE the comment after review: 
+# I was actually using this link "https://prod-dcd-datasets-cache-zipfiles.s3.eu-west-1.amazonaws.com/vz6d7tw87f-1.zip" which is copied from the 
+# "Download All" button from the website (https://data.mendeley.com/datasets/vz6d7tw87f/1). 
+# But output an error saying """/vsizip//tmp/tmp7puy6g90/vz6d7tw87f-1.zip' not recognized as a supported file format. 
+# It might help to specify the correct driver explicitly by prefixing the file path with '<DRIVER>:', e.g. 'CSV:path'""""
+SOURCES = "https://github.com/fiboa/data/files/15438544/LEM_dataset.zip"
```

Comment:
Shouldn't that be coming from the source?
https://data.mendeley.com/public-files/datasets/vz6d7tw87f/files/57c83c3f-b5a9-45f5-94f8-ac1df8fab923/file_downloaded

Reply:
+1 - I can't remember why I [attached it in a comment](https://github.com/fiboa/data/issues/25#issuecomment-2130118124), but this should be the source one.

Reply:
The reason why I didn't use the source link is because it didn't work. Here is the error output if I use the source link:
<img width="1523" alt="Screenshot 2024-07-26 at 11 08 15 AM" src="https://github.com/user-attachments/assets/ed8e3986-e108-46f5-a7cb-600427425565">

I dived into the `paths` variable in your `convert_utils.py` and printed them out, and the outputs looked similar for both `de_th` converter and my converter. But for some reason, it is not working. 


I also tried out the other link, which is done by right clicking the download button and selecting "copy link":
<img width="668" alt="Screenshot 2024-07-26 at 11 12 25 AM" src="https://github.com/user-attachments/assets/58ded55f-0eef-4e81-a7ea-d989ee0cf59d">
This will give me another link: `https://prod-dcd-datasets-cache-zipfiles.s3.eu-west-1.amazonaws.com/vz6d7tw87f-1.zip`

However, this link still doesn't work when I run the converter:
<img width="1526" alt="Screenshot 2024-07-26 at 11 13 51 AM" src="https://github.com/user-attachments/assets/2b056e84-5aee-4a9e-9df1-85682b0de663">

The only link that works is the link from GitHub. I think there might be some issues in the `convert_utils.py`? I am not sure.

Reply:
Could you try whether the following works, please?

```suggestion
SOURCES = {
    "https://github.com/fiboa/data/files/15438544/LEM_dataset.zip": "LEM_dataset.shp"
}
```

In ZIP files you sometimes need to explicitly specify which file from the package to load.

Reply:
It doesn't work:

```
`fiboa CLI 0.6.0 - Convert 'br_lem_ba'

Getting file(s) if not cached yet
Reading /tmp/tmpbmxsz9zu/LEM_dataset.shp into GeoDataFrame(s)
'/tmp/tmpbmxsz9zu/LEM_dataset.shp' not recognized as a supported file format. It might help to specify the correct driver explicitly by prefixing the file path with '<DRIVER>:', e.g. 'CSV:path'.`
```

I also tried:
```
SOURCES = {
    "https://data.mendeley.com/public-files/datasets/vz6d7tw87f/files/57c83c3f-b5a9-45f5-94f8-ac1df8fab923/file_downloaded": "LEM_dataset.shp"
}
```

And it is giving me the same error:
```
fiboa CLI 0.6.0 - Convert 'br_lem_ba'

Getting file(s) if not cached yet
Reading /tmp/tmpbw7r1nmn/LEM_dataset.shp into GeoDataFrame(s)
'/tmp/tmpbw7r1nmn/LEM_dataset.shp' not recognized as a supported file format. It might help to specify the correct driver explicitly by prefixing the file path with '<DRIVER>:', e.g. 'CSV:path'.
```

Reply:
May bad, I forgot to make it an array. The following works. I've already updated the file accordingly.

```py
SOURCES = {
    "https://github.com/fiboa/data/files/15438544/LEM_dataset.zip": ["LEM_dataset.shp"]
}
```